INN/umbrella-borderzine
# Homepage base layout with featured area and right rail sidebar

## Changes

This pull request makes the following changes:

- Creates a 1/3 2/3 homepage featured story area
   - Homepage Top Story prominence is 1/3
   - Homepage Featured prominence is 2/3
- Creates the `Homepage Sidebar` widget area, which appears to the right of the content under the Featured area
- `Homepage Bottom` widget area is now displayed to the left of the sidebar, still under the Featured area (might change in #6?)

![screencapture-borderzine-test-2019-09-10-14_39_48](https://user-images.githubusercontent.com/18353636/64641043-54779d80-d3d9-11e9-8b93-5b912f085f46.png)

## Why



For #2

## Testing/Questions

Features that this PR affects:

- Homepage template


Questions that need to be answered before merging:

- [ ] Do `Homepage Sidebar` and `Homepage Bottom` widget area names make sense?
- [x] What should happen with the sidebar on mobile?

Steps to test this PR:

1. Recreate `Main Sidebar` widgets in the `Homepage Sidebar` widget area
2. Verify `Homepage Bottom` widget area still has widgets
3. View the homepage and verify it looks ok
